### PR TITLE
Generation to support migration to Jakarta

### PR DESCRIPTION
**Pull request by Jad-el-khoury**
## Description

This PR depends on the Lyo PR https://github.com/eclipse/lyo/pull/500 to produce working code.
It is demonstrated by the generated code in the OSLC OP RefImpl in this PR https://github.com/oslc-op/refimpl/pull/198

## Checklist

- [x] This PR adds an entry to the CHANGELOG. _See https://keepachangelog.com/en/1.0.0/ for instructions. Minor edits are exempt. 
- [ ] This PR expects some manual changes to the generated code. The needed changes are detailed in the CHANGELOG entry.
- [x] This PR was tested with at least one code generation, resulting in a working OSLC server.
- [x] This PR does NOT break the user block code

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -21,6 +21,7 @@
 - Provide flexibility to for Store properties to be set via the Environment.
 - Changing the way Resource and ResourceProperty items are represented in dialogs. We include prefixes to make their identification easier.
 - Protect initServerProviders() method
+- Migrated from Javax to Jakarta packages, including upgrade to Jersey 3.1.5
```
Comment (berezovskyi) [inline CHANGELOG.md:24]: Would be good to:

1. Split into two entries.
2. Add some indication of a major breaking change (making the line bold and/or adding some emoji in the beginning like one of ❗️⚠️👉🚨‼️)
3. Move these to the top of the list.
4. Add entries under Removed:
   a. Removed support for Java 11.
   b. Removed support for Java EE and Jakarta EE 8.

**Review by berezovskyi — APPROVED** [review]
Looks good. See suggestion for Changelog improvement below.

File: CHANGELOG.md (hunk 1 above)
Comment (berezovskyi) [reply CHANGELOG.md:24]: https://en.wikipedia.org/wiki/Jakarta_EE reference for the versions.

Comment (Jad-el-khoury) [reply CHANGELOG.md]: Please see changes as suggested. ok?

**Pull request merged**